programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1918,6 +1918,7 @@ static void interrupt_handler(int sig) {
 #endif
 
 static void run(int argc, char *argv[]) {
+    setenv("UV_THREADPOOL_SIZE", "8", 0);
```

Comment:
I would not fix that here, let it be set at deployment

Reply:
I put this here to effectively increase the default to something reasonable for folks who run from the command line. The last argument to `setenv` prevents the call from doing anything if the variable is already present in the environment.

Reply:
realistically if DNS resolution is slow, it can fill an 8-worker pool just as well as a 4-worker one